tarorpc/taro.proto

```diff
@@ -135,17 +135,22 @@ message MintAssetRequest {
     */
     int64 amount = 4;
 
+    /*
+    The specific group key this asset should be minted with.
+    */
+    bytes group_key = 5;
```

Comment:
Are we not worried about breaking protos at this stage?

Reply:
I think nothing, including DB schema, is considered stabilized rn.